src/test/regress/output/multi_alter_table_statements.source

```diff
@@ -1066,6 +1066,64 @@ SELECT relname FROM pg_class WHERE relname LIKE 'test_table_1%';
 (0 rows)
 
 \c - - - :master_port
+-- verify logged info is propagated to workers when distributing the table
+CREATE TABLE logged_test(id int);
```

Comment:
I guess if you put the new tests to the end then you wouldn't have the other updates in this file. But it is a minor thing.

Reply:
I see what you mean. I tried to put the tests in a place that is somewhat consistent to the rest of the file.